extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js

```diff
@@ -41,7 +41,7 @@ define('ext.wikia.adEngine.slot.service.megaAdUnitBuilder', [
 			'LB': ['TOP_LEADERBOARD', 'MOBILE_TOP_LEADERBOARD'],
 			'MR': ['TOP_RIGHT_BOXAD'],
 			'PF': [
-				'MOBILE_PREFOOTER', 'BOTTOM_LEADERBOARD', 'MOBILE_BOTTOM_LEADERBOARD'
+				'MOBILE_PREFOOTER', 'BOTTOM_LEADERBOARD'
```

Comment:
What are our plans towards changing `'PF'` group in future? For example to `'BL'` ;)

Reply:
Right now we don't have to do this, but sure think we can groom on next occasion if we want to or should to do this.